src/modules/Builder/features/ReportConfigSetting/ReportConfigSetting.tsx

```diff
@@ -104,7 +103,7 @@ export const ReportConfigSetting = ({
   const { execute: postReportConfig } = useAsync(
     postReportConfigApi,
     () => {
```

Comment:
💅: here and below no need an anonymous function wrapper, right?
```
() => {
   handleSuccess();
}
```
use just `handleSuccess`

Reply:
Agreed, and it's what I was going to do, but it required a little rearranging of code. Event handler now has to be defined early, separately from where the other event handlers live. I'm fine with that if you are, so pushing this change!

Reply:
I'm ok with it, thank you!